src/porklock/pathing.clj

```diff
@@ -16,31 +15,36 @@
    ".irodsA"
    ".irodsEnv"])
 
+(defn- parse-exclude-file
+  "Returns a list of paths read from the file at the given `exclude-file-path`, or nil if the file does not exist."
+  [exclude-file-path delimiter]
+  (when (ft/exists? (str exclude-file-path))
+    (remove string/blank?
+            (string/split (slurp exclude-file-path)
+                          (re-pattern delimiter)))))
+
+(defn- paths-to-exclude
+  "Returns a list containing the static `relative-paths-to-exclude`
+   and the paths read from the file at the given `exclude-file-path`."
+  [exclude-file-path delimiter]
+  (concat relative-paths-to-exclude
+          (parse-exclude-file exclude-file-path delimiter)))
```

Comment:
Should we add the exclude file itself to the list of files to exclude? (or am I missing somewhere which is already doing that?)

Reply:
I was debating that, but I think we probably don't need to exclude the list itself. It may be good for debugging to have the list returned with other log files. Also, the current list of exclusions is exposed in the `docker-compose.yaml` file, which is also returned in the output logs, so we won't be exposing any new info.

If we decide to exclude the list itself, then the calling service can always add the name of the file it creates to the list.

Reply:
Seems reasonable. I thought I'd ask just in case!